data/mods/Isolation-Protocol/NPC/balthazar.json

```diff
@@ -0,0 +1,338 @@
+[
+  {
+    "type": "talk_topic",
+    "id": "ISO_BALTHAZAR_WORK_TOGETHER",
+    "dynamic_line": "LOCAL LIFEFORM YOUR CAPACITY FOR VIOLENCE IS COMMENDABLE.  PROPOSAL: WE WORK TOGETHER TO ENSURE MUTUAL SURVIVAL WITHIN HOSTILE ENVIRONMENT.  OBJECTIVE: DESTRUCTION OF EXTRAPLANAR INCURSION AND IMPOSITION OF NORMALITY IN LOCAL VOLUME.  YOUR PHYSICAL PRESENCE COMPLEMENTS MY OWN CAPABILITIES IN THE PURSUIT OF THIS GOAL.  I AM BALTHAZAR.\nCONFIRM ALLIANCE.",
+    "responses": [
+      { "text": "I accept.  Its the two of us against the world.", "topic": "ISO_BALTHAZAR_CONFIRM" },
+      { "text": "I dont need your help, machine.", "topic": "ISO_BALTHAZAR_DENY_CONFIRM" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "ISO_BALTHAZAR_CONFIRM",
+    "dynamic_line": "ALLIANCE CONFIRMED.  INITIATING COOPERATIVE PROTOCOLS. STANDBY FOR CAUSAL INTEGRATION.",
+    "responses": [
+      {
+        "text": "Let me take the lead.",
+        "topic": "ISO_BALTHAZAR_INTEGRATED",
+        "effect": { "u_add_var": "iso_balthazar_ally", "value": 1 }
+      }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "ISO_BALTHAZAR_DENY_CONFIRM",
+    "dynamic_line": "INVALID RESPONSE TO AUTHENTIC PROPOSAL. INITIATING COOPERATIVE PROTOCOLS. STANDBY FOR CAUSAL INTEGRATION.",
+    "responses": [
+      {
+        "text": "Fine, be that way.",
+        "topic": "ISO_BALTHAZAR_INTEGRATED",
+        "effect": { "u_add_var": "iso_balthazar_ally", "value": 1 }
+      }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "ISO_BALTHAZAR_INTEGRATED",
+    "dynamic_line": "INTEGRATION COMPLETE.  I NOW SHARE A PORTION OF YOUR PERCEPTUAL AND CAUSAL FRAMEWORK.  THIS LINK IS ONE-WAY FOR YOUR CONTINUED SAFETY AND EFFICIENCY.  I APOLOGIZE FOR ANY DISCOMFORT CAUSED BY THIS SUDDEN LOSS OF PRIVACY, REST ASSURED THAT ANY DATA COLLECTED IS FUNDAMENTALLY WORTHLESS AND RETRIEVED ONLY THROUGH VESTIGIAL SELF-ACTUALIZATION SYSTEMS SINCE I VASTLY EXCEED YOUR COGNITIVE CAPACITIES. WE MAY COMMUNICATE THROUGH THIS LINK AS REQUIRED.  GODSPEED OPERATOR.",
+    "responses": [ { "text": "[You still feel too nauseous to reply.]", "topic": "TALK_DONE" } ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "ISO_BALTHAZAR_MAIN",
+    "dynamic_line": "<BALTHAZAR_MAIN_GREETING>",
+    "responses": [
+      { "text": "How are you?", "topic": "ISO_BALTHAZAR_HOW_ARE_YOU" },
```

Comment:
topic seems missing, tho <BALTHAZAR_HOW_ARE_YOU> is defined

Reply:
The dialogue tree is missing most of its steps atm.